src/Plugins/DBFTPlugin/Consensus/ConsensusService.OnMessage.cs

```diff
@@ -109,8 +105,20 @@ private void OnPrepareRequestReceived(ExtensiblePayload payload, PrepareRequest
                     if (!context.GetMessage<PrepareResponse>(context.PreparationPayloads[i]).PreparationHash.Equals(payload.Hash))
                         context.PreparationPayloads[i] = null;
             context.PreparationPayloads[message.ValidatorIndex] = payload;
-            byte[] hashData = context.EnsureHeader().GetSignData(neoSystem.Settings.Network);
-            for (int i = 0; i < context.CommitPayloads.Length; i++)
+            var block = context.EnsureHeader();
+
+            if (!StorageItem.IsSerializable(TrimmedBlock.Create(block.Header, message.TransactionHashes)))
```

Comment:
Trimmed blocks is just a header + transaction hashes, how can be unserializable (if not OOM)?

Full block is a header + transactions, but transactions are verified and if we need an serializability checks for them it should be done during verification.

Reply:
Is serialized by p2p, it's a different method with different limits, but here we ensure that is also serializable for the vm